packages/react-core/src/components/Toolbar/examples/Toolbar.md

```diff
@@ -1237,34 +1224,13 @@ class ToolbarStacked extends React.Component {
       </React.Fragment>
     );
 
-    const iconButtonGroupItems = (
-      <React.Fragment>
-        <ToolbarItem>
-          <Button variant="plain" aria-label="edit">
-            <EditIcon />
-          </Button>
-        </ToolbarItem>
-        <ToolbarItem>
-          <Button variant="plain" aria-label="clone">
-            <CloneIcon />
-          </Button>
-        </ToolbarItem>
-        <ToolbarItem>
-          <Button variant="plain" aria-label="sync">
-            <SyncIcon />
-          </Button>
-        </ToolbarItem>
-      </React.Fragment>
-    );
-
     const firstRowItems = (
       <React.Fragment>
         <Toolbar>
           <ToolbarContent>
             <ToolbarToggleGroup toggleIcon={<FilterIcon />} breakpoint="lg">
               {toggleGroupItems}
             </ToolbarToggleGroup>
-            <ToolbarGroup variant="icon-button-group">{iconButtonGroupItems}</ToolbarGroup>
```

Comment:
why did we remove this?

Reply:
because it was making the example too wide for patternfly.org